webpack/assets/javascripts/react_app/components/SettingsTable/SettingsTableHelpers.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+
+import { Tooltip, OverlayTrigger } from 'patternfly-react';
+
+import { translate as __ } from '../../common/I18n';
+
+import { deepPropsToCamelCase } from '../../common/helpers';
+
+export const withTooltip = Component => componentProps => {
+  const { tooltipId, tooltipText, ...rest } = componentProps;
+
+  return (
+    <OverlayTrigger
+      overlay={<Tooltip id={tooltipId}>{tooltipText}</Tooltip>}
+      trigger={['hover', 'focus']}
+      placement="top"
+      rootClose={false}
+    >
+      <span>
+        <Component {...rest} />
+      </span>
+    </OverlayTrigger>
+  );
+};
+
+export const arraySelection = setting => {
+  const { selectValues } = setting;
+
+  if (!Array.isArray(selectValues)) {
+    return null;
+  }
+  return deepPropsToCamelCase(selectValues);
```

Comment:
I don't really get this :thinking:

Reply:
It transforms keys of the select options to proper case:

```js
console.log(selectValues)

[
  { value: null, label: 'Select an owner' },
  {
    group_label: 'Users',
    children: [
      { label: 'Admin', value: '4-Users' }
    ],
  },
 {
    group_label: 'Usergroups',
    children: [
      { label: 'Default Usergroup', value: '5-Usergroups' }
    ],
  }
]
```

Reply:
Ah! I see `group_label => groupLabel` ^_^